dashboard/public/openapi.yaml

```diff
@@ -23,10 +20,1621 @@ info:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    $ref: '#/components/schemas/Release'
+        '401':
+          description: Unauthorized
+    delete:
+      tags:
+        - kubeops
+      summary: 'DeleteRelease'
+      description: ''
+      operationId: 'DeleteRelease'
+      parameters:
+        - name: cluster
+          in: path
+          required: true
+          schema:
+            type: string
+            default: default
+          description: Cluster name
+        - name: namespace
+          in: path
+          required: true
+          schema:
+            type: string
+            default: default
+          description: Namespace name
+        - name: releaseName
+          in: path
+          required: true
+          schema:
+            type: string
+            default: default
+          description: Release name
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: string
+        '401':
+          description: Unauthorized
+    put:
+      tags:
+        - kubeops
+      summary: 'OperateRelease'
+      description: ''
+      operationId: 'OperateRelease'
+      parameters:
+        - name: cluster
+          in: path
+          required: true
+          schema:
+            type: string
+            default: default
+          description: Cluster name
+        - name: namespace
+          in: path
+          required: true
+          schema:
+            type: string
+            default: default
+          description: Namespace name
+        - name: releaseName
+          in: path
+          required: true
+          schema:
+            type: string
+            default: default
+          description: Release name
+        - name: action
+          in: query
+          schema:
+            type: string
+            enum: ["upgrade","rollback"]
+            default: upgrade
+          description: Action to perform
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    $ref: '#/components/schemas/Release'
+        '401':
+          description: Unauthorized
+  
 components:
+  schemas:
+    # pkg/proxy/proxy.go
+    AppOverview:
+      type: object
+      properties:
+        nareleaseNameme:
+          type: string
+        version:
+          type: string
+        namespace:
+          type: string
+        icon:
+          type: string
+        status:
+          type: string
+        chart:
+          type: string
+        chartMetadata:
+          $ref: '#/components/schemas/ChartMetadata'
+    # /pkg/mod/k8s.io/helm@v2.16.0+incompatible/pkg/proto/hapi/chart/metadata.pb.go
```

Comment:
Why is this based on Helm 2.6 instead of Helm 3 (as in https://github.com/kubeapps/kubeapps/pull/2291/files#diff-da58234bb539b4febe5d392f206d2fd4025fb7c76f89c71653c4d590bcd72f2bR1474)?